x-pack/plugins/secops/public/components/page/hosts/host_summary/index.tsx

```diff
@@ -101,7 +101,7 @@ export const createDraggable = (
   endDate: number,
   relativeFields: string[]
 ) => {
-  return summaryValue == null ? (
+  return summaryValue == null || summaryValue === '' ? (
```

Comment:
Nice catch on this fix and the added tests above. Something to pay attention to for other components now that packetbeat data flowing.

Reply:
As discussed this is now handled by the empty string value component.